packages/teams-components/src/components/Button/Button.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import {
+  useButton_unstable,
+  useButtonStyles_unstable,
+  renderButton_unstable,
+  type ButtonProps as ButtonPropsBase,
+  Tooltip,
+} from '@fluentui/react-components';
+import { validateIconButton } from './validateIconButton';
+import { type StrictCssClass, validateStrictClasses } from '../../strictStyles';
+import { type StrictSlot } from '../../strictSlot';
+
+export interface ButtonProps
+  extends Pick<
+    ButtonPropsBase,
+    | 'aria-label'
+    | 'aria-labelledby'
+    | 'aria-describedby'
+    | 'size'
+    | 'children'
+    | 'disabled'
+    | 'disabledFocusable'
+  > {
+  appearance?: 'transparent' | 'primary';
+  className?: StrictCssClass;
+  icon?: StrictSlot;
+  onClick?: React.MouseEventHandler<HTMLButtonElement>;
+  tooltip?: StrictSlot;
+}
+
+export const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
+  (userProps, ref) => {
+    if (process.env.NODE_ENV !== 'production') {
+      validateProps(userProps);
+    }
+
+    const { className, icon, tooltip, ...restProps } = userProps;
+    const props: ButtonPropsBase = {
+      ...restProps,
+      className: className?.toString(),
+      iconPosition: 'before',
+      icon,
+    };
+
+    let state = useButton_unstable(props, ref);
+    state = useButtonStyles_unstable(state);
+
+    const button = renderButton_unstable(state);
+
+    if (tooltip) {
+      return (
+        <Tooltip content={tooltip} relationship="label">
```

Comment:
We discussed to have only controlled components: it would be a funny problem if somebody wants to control this `Tooltip`.

Reply:
hmm let's call it `title` them as you suggested, to remove the notion that it's a tooltip that can be controlled - people can always compose a Tooltip and and use the title prop